core/trino-main/src/main/java/io/trino/operator/exchange/LocalExchange.java

```diff
@@ -13,6 +13,7 @@
  */
```

Comment:
add a comment that `PageReference` is no longer needed after removing broadcast exchange

Reply:
Should I add it to the commit message? Not sure if it makes sense to keep mentions of the `PageReference` in the codebase, since the PageReference itself is going away